toxcore/friend_connection.c

```diff
@@ -403,6 +406,10 @@ static int handle_status(void *object, int number, uint8_t status, void *userdat
     if (status_changed) {
         unsigned int i;
```

Comment:
Move `i` below the `if` or even into the `for` init decl.

Reply:
Agreed, but this has nothing to do with this PR.

Reply:
It does, because you introduce new lines between the declaration and the `for`.